<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Planning_Page</class>
 <widget class="QDialog" name="Planning_Page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QLabel" name="SelectedPath_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Your Selected Path</string>
   </property>
  </widget>
  <widget class="QLabel" name="CustomPath_label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Custom Path Selection</string>
   </property>
  </widget>
  <widget class="QLabel" name="To_label_2">
   <property name="geometry">
    <rect>
     <x>621</x>
     <y>100</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QLabel" name="From_label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QLabel" name="To_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>97</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QComboBox" name="To_ComboBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>73</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="From_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>97</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QComboBox" name="From_ComboBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>73</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="Distance_label">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>97</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="Distances_label">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Distances</string>
   </property>
  </widget>
  <widget class="QLabel" name="units_label">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>80</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>miles</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Distance_TextBrowser_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>70</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="EqualSign_label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>76</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Food_TextBrowser">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>160</y>
     <width>221</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="BuyFood_PushButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>500</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Buy Food</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Food_ComboBox">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>120</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QSpinBox" name="Quantity_SpinBox">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>120</y>
     <width>43</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="Total_label">
   <property name="geometry">
    <rect>
     <x>864</x>
     <y>530</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Total_label_2">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>530</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>331</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTextBrowser" name="StartCity_TextBrowser">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="EndCityList">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="DistanceList">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>331</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTextBrowser" name="Route_TextBrowser">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="Route_TextBrowser_2">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="Distance_TextBrowser">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>490</y>
     <width>111</width>
     <height>67</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="Save_Route_pushButton">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save Route</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ClearPath_PushButton">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Clear Path</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>121</width>
     <height>103</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="ChoosePath_pushButton">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Choose Path</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Paris_PushButton">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start at Paris</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="London_PushButton">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start at London</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
